modules/gmc-agent/src/main/java/org/gridgain/utils/QueryUtils.java

```diff
@@ -123,7 +170,7 @@ public static SqlFieldsQuery prepareQuery(QueryArgument arg) {
             Object[] row = new Object[sz];
 
             for (int i = 0; i < sz; i++)
-                row[i] = VisorQueryUtils.convertValue(next.get(i));
+                row[i] = convertValue(next.get(i));
```

Comment:
Lets return JSON for key and value.
See: org.apache.ignite.internal.processors.rest.protocols.http.jetty.GridJettyObjectMapper#IGNITE_BINARY_OBJECT_SERIALIZER

As example of JSON serializer for binary objects.

Reply:
Let's do it in separate ticket. GG-24424